### ISSUE TITLE
Video Link is Dead

### ISSUE THREAD

**Issue by hcovitz** [MicrosoftDocs/azure-devops-docs#11942]
[Enter feedback here]
Ironically, because here I am now, on GitHub, the link provided in: 
"Azure DevOps Services supports integration with GitHub.com and GitHub Enterprise Server repositories. Azure DevOps Server supports integration with GitHub Enterprise Server repositories. For more information, see the following video, [Using GitHub with Azure DevOps](https://msit.microsoftstream.com/video/77d07845-8091-48df-8f56-adc1e5b4d8eb?referrer=https:%2F%2Fmicrosoft.sharepoint.com%2F)." 
not only takes you to a Streams realm where you are not allowed to use a "personal account" (doesn't say but implies you need a institutional account), but, get this, the notice that the "video could not be found. it may have been removed.".
Sweet. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2197a6a8-ecda-b647-6cbf-c08defccc54b
* Version Independent ID: c66fa15d-cb16-b734-d73b-701acf368f95
* Content: [Plan, code, collaborate, ship applications - Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/user-guide/what-is-azure-devops?view=azure-devops)
* Content Source: [docs/user-guide/what-is-azure-devops.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/user-guide/what-is-azure-devops.md)
* Product: **devops**
* Technology: **devops-new-user**
* GitHub Login: @chcomley
* Microsoft Alias: **chcomley**

**Comment by TimShererWithAquent** [comment]
@hcovitz - Can you try again? I can follow the link. The video plays. I don't get that message.

If you try again and get the same message, could you do a screen cap?

Also, when you say that you're on GitHub, do you mean that you're trying to access it from here? MicrosoftDocs/azure-devops-docs/blob/main/docs/user-guide/what-is-azure-devops.md Links are only intended to work from the Docs site.

**Comment by hcovitz** [comment]
I was trying to access from this article:
https://docs.microsoft.com/en-us/azure/devops/user-guide/what-is-azure-devops?view=azure-devops
... which is how I arrived at thsi "issues" section on GitHub (since at bottom of that doc.microsoft.com page if you click "Feedback for this Page" it takes you to this GitHub repo.
Screen captures from different environments, including one from a company that happened to have an enterprise license as well as a MS Streams setup (which is how I got the "video could not be found" link, otherwise I'm stopped cold by either "no personal account sign in" verbiage or, using another enterprise that had a MS license but not their MS Streams setup appropriately? who know?  but I get the invitation to, I can only guess, setup their MS Teams ?).

![msFail1c](https://user-images.githubusercontent.com/8238705/152617482-9c101c77-78d3-49ce-adaa-aa1bdea968eb.png)
![msFail1](https://user-images.githubusercontent.com/8238705/152617485-35769b88-7c32-4c67-ab14-a5e1c35288a3.PNG)
![msFail1b](https://user-images.githubusercontent.com/8238705/152617486-6d9f4ca6-5bed-4848-ad89-e9f4aa38c67a.PNG)
![msFail1a](https://user-images.githubusercontent.com/8238705/152617487-978a4489-6526-45cb-a3bb-1aba78f6ac1a.PNG)
![msLinkFail2](https://user-images.githubusercontent.com/8238705/152617488-4c5b2959-d3f6-4fa0-9ee9-92717ed2c3d5.png)

**Comment by hcovitz** [comment]
The fact that this is intermittent, and that apparently some people can access it, is significantly more disturbing.

**Comment by TimShererWithAquent** [comment]
@hcovitz Thanks for taking the extra time on this. We'll see what's going on -

**Comment by chcomley** [comment]
Hi @hcovitz, I've looked into this issue and determined that the linked video appears to be internal to Microsoft and was added a few years ago. I'm sorry for the confusion and frustration this may have caused. I've replaced the video link with a link to the following article: [Azure DevOps and GitHub integration overview](https://docs.microsoft.com/en-us/azure/devops/cross-service/github-integration?view=azure-devops). Thanks so much for your feedback. This issue will automatically close once the update goes live. Feel free to @mention me if you have any further questions. Thanks again!
